ARM-wvd-templates/HCI/QuickDeploy/CreateHciHostpoolQuickDeployTemplate.json

```diff
@@ -0,0 +1,298 @@
+      "appGroupName": "[concat(variables('hostPoolName'),'-DAG')]",
+      "appGroupResourceId": "[createArray(resourceId('Microsoft.DesktopVirtualization/applicationgroups/', variables('appGroupName')))]",
+      "hostpoolRequiredProps": {
+        "hostpoolType": "Pooled",
+        "maxSessionLimit": "99999",
+        "loadBalancerType": "BreadthFirst",
+        "validationEnvironment": "[parameters('validationEnvironment')]",
+        "preferredAppGroupType": "Desktop",
+        "ring": null,
+        "registrationInfo": {
+          "expirationTime": "[variables('tokenExpirationTime')]",
+          "registrationTokenOperation": "Update"
+        }
+      }
+    },
+    "resources": [
+      {
+        "name": "[parameters('hostPoolName')]",
+        "apiVersion": "2019-12-10-preview",
+        "type": "Microsoft.DesktopVirtualization/hostpools",
+        "location": "[parameters('location')]",
+        "tags": "[parameters('tags')]",
+        "properties": "[variables('hostpoolRequiredProps')]"
+      },
+      {
+        "name": "[variables('appGroupName')]",
+        "apiVersion": "2019-12-10-preview",
+        "type": "Microsoft.DesktopVirtualization/applicationgroups",
+        "location": "[parameters('location')]",
+        "tags": "[parameters('tags')]",
+        "properties": {
+          "hostpoolarmpath": "[resourceId('Microsoft.DesktopVirtualization/hostpools/', parameters('hostPoolName'))]",
+          "friendlyName": "Default Desktop",
+          "description": "Desktop Application Group created through the Hostpool Wizard",
+          "applicationGroupType": "Desktop"
+        },
+        "dependsOn": [
+          "[resourceId('Microsoft.DesktopVirtualization/hostpools/', parameters('hostPoolName'))]"
+        ]
+      },
+      {
+        "apiVersion": "2018-05-01",
+        "name": "[concat('Workspace-linkedTemplate-', parameters('deploymentId'))]",
+        "type": "Microsoft.Resources/deployments",
+        "properties": {
+          "mode": "Incremental",
+          "template": {
+            "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+            "contentVersion": "1.0.0.0",
+            "resources": [
+              {
+                "apiVersion": "2019-12-10-preview",
+                "name": "[parameters('workspaceName')]",
+                "type": "Microsoft.DesktopVirtualization/workspaces",
+                "location": "[parameters('location')]",
+                "properties": {
+                  "applicationGroupReferences": "[variables('appGroupResourceId')]"
+                }
+              }
+            ]
+          }
+        },
+        "dependsOn": [
+          "[resourceId('Microsoft.DesktopVirtualization/applicationgroups/', variables('appGroupName'))]"
+        ]
+      },
+      {
+        "apiVersion": "2018-05-01",
+        "name": "[concat('hci-addVm-linkedTemplate-', parameters('deploymentId'))]",
```

Comment:
why this has name add instead of create

Reply:
it was originally create, but went over the character limit so i changed

Reply:
FYI for fun, StackHCI calls all of their stuff PutVm... in their examples .   they don't use add or create :)